docs/src/examples/components/Menu/Variations/MenuExampleVerticalPointingEnd.shorthand.tsx

```diff
@@ -8,7 +8,7 @@ const items = [
 ]
 
 const MenuExampleVerticalPointingEnd = () => (
-  <Menu defaultActiveIndex={0} items={items} vertical pointing="end" />
+  <Menu defaultActiveIndex={0} items={items} vertical pointing="end" primary />
```

Comment:
This looks like a breaking change, is `primary` required now to get the same styling?

Reply:
Previously, we discussed this as I remember:
- replace `type` with `primary` and `secondary` props, #429
- remove `primary` and `secondary` from `Divider` as it doesn't applies any styling, #558
- on components like `Menu`, `primary` and `secondary` can affect styling, too: https://react.semantic-ui.com/collections/menu/#types-secondary

May be, in this case we should not expose `primary` and `secondary` as colors to avoid confusions.

Reply:
I talked with @levithomason yesterday about this and he also thinks the `primary`, `secondary` prop for Menu should be removed. I like the solution of just using `color` prop for all color style updates.

Reply:
not relevant anymore, see #756
`primary` is not needed as a boolean prop any longer